### ISSUE TITLE
Can't import knxproj file from ETS 6.1 and 6.2

### ISSUE THREAD

**Issue by lgusewski** [XKNX/xknxproject#389]
I had imported an older knxproj file in August 2023 from ETS 6.1.5686.0. Today i wanted to import an newer version from ETS 6.2 to Home Assistant 2024.4.1 and got an error message that the file is to large (1,5 MB). I tried to reload the older one from last year and got the same error message (1,4 MB). Perhaps something broken in the import modul.

**Comment by farmio** [comment]
Hi 👋! 
Have you had a look at https://github.com/XKNX/xknxproject/issues/258 and its linked issues?

**Comment by lgusewski** [comment]
Hello, I did not saw this before. A http connection solved the problem, so the issue is wrong, not the ETS format is the problem, it was the proxy connection, too.

But in this case I did not understand why the official Add-on for nginx proxy did not have the necessary presets as default, so that the same max size is possible for http and https connections.

Where is the right place, to trigger this as an change for an upcoming release?
